django_statsd/clients/request_aggregate.py

```diff
@@ -0,0 +1,24 @@
+from statsd.client import StatsClient
+
+try:
+    from django.utils._threading_local import local
+except ImportError:
+    from threading import local
+
+class AggregateStatsMixin(object):
+    aggregate_request_stats = local()
+
+    def timing(self, stat, delta, rate=1):
+        super(AggregateStatsMixin, self).timing(stat, delta, rate)
+        # view stats only happen once per request so no aggregation necessary
+        if stat.startswith('view.'):
```

Comment:
Could you move this out to a class variable or something so that it can be overrided?

There might be non-view timings that aggregate info would be useful for, and it seems adding that would allow us to do so easily.